ports/esp32/boards/ESP32_S2_WROVER/mpconfigboard.h

```diff
@@ -0,0 +1,12 @@
+#define MICROPY_HW_BOARD_NAME "ESP32-S2-WROVER"
+#define MICROPY_HW_MCU_NAME "ESP32-S2"
+
+#define MICROPY_PY_BLUETOOTH                (0)
+#define MICROPY_HW_ENABLE_SDCARD            (0)
+
+#define MICROPY_HW_I2C0_SCL (9)
+#define MICROPY_HW_I2C0_SDA (8)
+
+#define MICROPY_HW_SPI1_MOSI (35)  // SDO
+#define MICROPY_HW_SPI1_MISO (37)  // SDI
+#define MICROPY_HW_SPI1_SCK (36)
```

Comment:
There is also SPI(2), with defaults of pins 18/19/23.  Is that correct for this board?

Reply:
No 18/19/23 is not correct for the ESP32-S2, I have updated the pins for both SPI's according the the module datasheet.